hudi-cli/src/main/java/org/apache/hudi/cli/commands/CommitsCommand.java

```diff
@@ -96,7 +98,15 @@ public String rollbackCommit(@CliOption(key = {"commit"}, help = "Commit to roll
       throws Exception {
     HoodieActiveTimeline activeTimeline = HoodieCLI.getTableMetaClient().getActiveTimeline();
     HoodieTimeline timeline = activeTimeline.getCommitsTimeline().filterCompletedInstants();
-    HoodieInstant commitInstant = new HoodieInstant(false, HoodieTimeline.COMMIT_ACTION, commitTime);
+    HoodieTableType tableType = HoodieCLI.getTableMetaClient().getTableType();
+    HoodieInstant commitInstant;
```

Comment:
A MergeOnRead table has both DELTA_COMMIT and COMMIT. The best way to do this would be : 

instant = activeTimeline.getCommitsTimeline().filterCompletedInstants().filter(instant.getTimestamp.equals(commitTime)
Use that instant (which should have the action populated) and you can remove the if else..